src/main/java/ch/epfl/biop/ij2command/Rois2MeasurementMap.java

```diff
@@ -35,7 +35,22 @@
     @Parameter
     RoiManager rm ;
 
-    @Parameter ( label="Measure", choices = {"Area", "Angle", "AngleVert","AR", "Circ.", "Major","Minor","Mean","Median","Mode","Min","Max", "Perim.","Pattern"})
+    @Parameter(label = "Measure", choices = {   "Area",
+                                                "Angle",
+                                                "AngleVert",
+                                                "AR",
+                                                "Circ.",
+                                                "Major",
+                                                "Minor",
+                                                "Mean",
+                                                "Median",
+                                                "Mode",
+                                                "Min",
+                                                "Max",
+                                                "Perim.",
+                                                "Pattern",
+                                                "xCenterOfMass",
+                                                "yCenterOfMass"})
     String column_name;
 
     @Parameter ( label="If Pattern, please specify a regex capture group,\n(will take first group, must be numerical)", required=false )
```

Comment:
Put an example of pattern in the description annotation field - this shows up when the user has its mouse over the parameter name

Reply:
solved